File: code_samples/kotlin/testing_fixtures.kt
/**
 * Testing: Fixtures
 * AI/ML Training Sample
 */

class Fixtures {
    var data: String = ""
        private set
    
    fun process(input: String) {
        data = input
    }
    
    fun getData(): String = data
    
    fun validate(): Boolean = data.isNotEmpty()
}

fun main() {
    val instance = Fixtures()
    instance.process("example")
    println("Data: ${instance.getData()}")
    println("Valid: ${instance.validate()}")
}

File: code_samples/csharp/testing_fixtures.cs
using System;

/// <summary>
/// Testing: Fixtures
/// AI/ML Training Sample
/// </summary>
public class Fixtures
{
    private string data;
    
    public Fixtures()
    {
        this.data = string.Empty;
    }
    
    public void Process(string input)
    {
        this.data = input;
    }
    
    public string GetData()
    {
        return this.data;
    }
    
    public bool Validate()
    {
        return !string.IsNullOrEmpty(this.data);
    }
    
    public static void Main(string[] args)
    {
        var instance = new Fixtures();
        instance.Process("example");
        Console.WriteLine($"Data: {instance.GetData()}");
        Console.WriteLine($"Valid: {instance.Validate()}");
    }
}
